prow/cluster/starter.yaml

```diff
@@ -0,0 +1,543 @@
+---
```

Comment:
I would like to know, from which revision this file is copied.
Additionally, please document how this file differs from a original one.

Reply:
Done